sensorExamples/joystickREADME.md

```diff
@@ -0,0 +1,24 @@
+# Demo: Adafruit 2-Axis Thumb Joystick
+
+The [Analog 2-axis Thumb Joystick](http://www.adafruit.com/products/512) 
+allows you to easily mount a PSP/Xbox-like thumb joystick to your project. 
+Using analog pins, the BeagleBone will read and determine both the X and Y axis. 
+The joystick also includes an extra digital input that will let you read the switch.
+
+*The console will output both the x-axis and y-axis with readings from 0-100, 
+with (50,50) being the center location.*
+
+![alt text](joystick.png "Joystick")
+
+## Build and execute instructions
+* Connect the following pins
+
+Joysitck | Pin | Bone  | Pocket
+-------- | --- | ----- | ------
+VCC | 1.8V | P9_32 | P1_18
+GND | GND  | P9_34 | P1_17
+HOR | AIN5 | P9_36 | P2_35
+VER | AIN3 | P9_38 | P1_25
+SEL | GPIO | P9_42 | P1_35
```

Comment:
Specify GPIO and use GPIOx specifier in the code to be board generic, but GPIOs will have different numbers on AI.

Reply:
Why use GPIO numbers rather than pin numbers?  
Can you use GPIO numbers in bonescript?

(It turns out the joystick.js code never uses the SEL.)